docs/guide/essentials/history-mode.md

```diff
@@ -19,6 +19,8 @@ Not to worry: To fix the issue, all you need to do is add a simple catch-all fal
 
 ## Example Server Configurations
 
+**Note**: The following examples assume you are serving your app from the root folder. If you deploy to a subfolder, see [the `publicPath` option of Vue CLI](https://cli.vuejs.org/config/#publicpath) and the related [`base` property of the router](https://router.vuejs.org/api/#base).
```

Comment:
I think we should give a bit more information like: you should use the publicPath and the related... and adapt the rewrite rules to point to `/base` instead of `/`

Reply:
Fair point, I was wondering about the right level of verbosity. Does the new wording work?